modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/VersionedRowStore.java

```diff
@@ -1,658 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Well, It wasn't expected that you will remove VersionRowStore because it should be used as a reference implementation for some logic, e.g. snapshot flushing. Thus, as you've already remove given class, please move snapshot related logic to existing PartitionListener.

Reply:
Added snapshot related code to the partition listener.